### ISSUE TITLE
Accessibility: SparksieBOT "Live!" button is not selectable with keyboard

### ISSUE THREAD

**Issue by spark-c** [spark-c/spark-c.github.io#4]
Since this button is not technically a link like the others, it does not receive keyboard focus.

Perhaps wrapping it in an anchor with an onClick handler with `e.preventDefault()`? This may be poor practice; research required.
